READMEv2.md

```diff
@@ -0,0 +1,161 @@
+  let u2 = oneDRotation({ x: player2.velocityX, y: player2.velocityY }, rotatedAngle);
+
+  let v1 = { x: u2.x, y: u1.y };
+  let v2 = { x: u1.x, y: u2.y };
+
+  let finalVel1 = oneDRotation(v1, -rotatedAngle);
+  let finalVel2 = oneDRotation(v2, -rotatedAngle);
+
+  player1.velocityX = finalVel1.x;
+  player1.velocityY = finalVel1.y;
+
+  player2.velocityX = finalVel2.x;
+  player2.velocityY = finalVel2.y;
+
+  let xPow = Math.pow(player2.coordX - player1.coordX, 2);
+  let yPow = Math.pow(player2.coordY - player1.coordY, 2);
+  let distance = Math.sqrt(xPow + yPow);
+
+};
+
+const oneDRotation = (velocity, angle) => {
+  const rotatedVelocities = {
+      x: velocity.x * Math.cos(angle * (180/Math.PI)) - velocity.y * Math.sin(angle * (180/Math.PI)),
+      y: velocity.x * Math.sin(angle * (180/Math.PI)) + velocity.y * Math.cos(angle * (180/Math.PI))
+  };
+
+  return rotatedVelocities;
+}
+```
+
+<a name="projectile-logic">
+  <h2>
+    <img src="https://raw.githubusercontent.com/MoistCode/ElementBlaster/gh-pages/assets/favicons/favicon-16x16.png">
+      Sample Projectile Logic
+  </h2>  
+</a>
+
+  Projectile logic was a tad bit easier as it was limited to only one of the four cardinal directions. By providing a velocity and lifecycle, the bullets were able to become recycled so players don't get a screen full of flying bullets...although that might be a difficulty in the future for the crazy folks. The colors correspond to the elements and depending on what element the player is on, they will only shoot that type of element.
+
+```javascript 
+      this.context.beginPath();
+        this.context.rect(
+          this.posX - (13/2),
+          this.posY + 40,
+          13,
+          13
+        );
+        switch(this.element) {
+          case 'Fire':
+            this.context.fillStyle = 'red';
+            break;
+          case 'Earth':
+            this.context.fillStyle = '#7B1803';
+            break;
+          case 'Lightning':
+            this.context.fillStyle = '#F5EE10';
+            break;
+          case 'Water':
+            this.context.fillStyle = 'blue';
+            break;
+        }
+        this.context.fill();
+        this.context.closePath();
+
+        this.context.beginPath();
+        this.context.arc(this.posX, this.posY + 52, 7, 0, Math.PI);
+        this.context.fillStyle = 'white'
+        this.context.fill();
+        this.context.closePath();
+
+        this.posY += this.velocity;
+        this.lifeline -= 1;
+```
+
+<a name="project-direction">
+  <h2>
+    <img src="https://raw.githubusercontent.com/MoistCode/ElementBlaster/gh-pages/assets/favicons/favicon-16x16.png">
+      Project Direction
+  </h2>  
+</a>
+
+* Create gifs on my desktop because my laptop sucks
+* Diagonal blasts
+* Boss fights 
+* Ability to counter elemental bullets by changing player element
+* Power-ups
+* Different game music options
+* Scoring system
```

Comment:
What

Reply:
I just realised that's the first commit, not the latest.

Reply:
Yeah I took it from a previous README format.